OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -69,6 +70,9 @@ public sealed class WeaponInfo
 		[Desc("The minimum range the weapon can fire.")]
 		public readonly WDist MinRange = WDist.Zero;
 
+		[Desc("Does this weapon aim at the target's center regardless of other targetable offsets?")]
+		public readonly bool AttackTargetCenter = false;
```

Comment:
How about `TargetActorCenter`?  The "attack" is implicit, and this only applies to actors.